.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "codelabz-dev"
+    "default": 'buoyant-country-311408'
```

Comment:
Ok, Now i'm seeing the issue here. let's ignore the firebaserc file. Revert this change back.

Reply:
can I do this like, change firebaserc to the default 'codelabz-dev' and push the commit ,  then I can add this file to gitignore and push. I think it will solve the problem , our main codebase has default value as "codelabz-dev" , and if anyone change this file it will not effect our main codebase.

Reply:
@shehand  changed as mentioned above.